src/styles/blocks/catalog.scss

```diff
@@ -2,8 +2,8 @@
 .catalog {
   display: flex;
   flex-wrap: wrap;
-  padding: 50px 40px;
-  width: 1024px;
+  padding: 50px 0;
+  max-width: $cards * $width-card + $right-margin-card * ($cards - 1);
   margin: 0 auto;
```

Comment:
```suggestion
```

Not the right way to center the block. BEM says that the block doesn't determine its positioning. Instead, create the `main` block and add the necessary styling there.